### DISCUSSION TITLE
Workspace Admins should be able to list all collections via the API

### DISCUSSION THREAD

**Discussion by blulightspecial** [outline/outline#5819] [Ideas]
Currently when a workspace admin calls `collections.list` via the api, they only get returned a list of collections that they or their groups have explicitly been added to either as members or admins. It would be helpful if for workspace admins they were instead given a list of all collections regardless of whether they have access or not when they call the `collections.list` endpoint. 

There is already a pathway to this info in the GUI. As a workspace admin, I can see all collections including ones that I'm not a member of in the workspace settings menu.

Granting such access through the API would enable handy things like:
-  programatic exports of the entire workspace
  - currently only repos that the admin is a member of get exported
- Programatic adding of an admin to all collections

**Comment by tommoor** [comment]
You can use the (perhaps oddly named) parameter `includeListOnly=true` on the `collections.list` API request as an admin to also return the other collections
